### ISSUE TITLE
assembly-versioning-format throws a SerializationException

### ISSUE THREAD

**Issue by timoch** [GitTools/GitVersion#1406]
I have a configuration that looks like this

```yaml
commit-message-incrementing: Disabled
assembly-versioning-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}'
assembly-file-versioning-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}'
```

It looks like I correctly follow the [documentation](http://gitversion.readthedocs.io/en/latest/configuration/)
but I receive the following :

```
ERROR [05/04/18 9:36:32:82] An unexpected error occurred:
YamlDotNet.Core.YamlException: (Line: 1, Col: 1, Idx: 0) - (Line: 1, Col: 1, Idx: 0): Exception during deserialization ---> System.Runtime.Serialization.SerializationException: Property 'assembly-versioning-format' not found on type 'GitVersion.Config'.
   at YamlDotNet.Serialization.TypeInspectors.TypeInspectorSkeleton.GetProperty(Type type, Object container, String name, Boolean ignoreUnmatched)
   at YamlDotNet.Serialization.NodeDeserializers.ObjectNodeDeserializer.YamlDotNet.Serialization.INodeDeserializer.Deserialize(EventReader reader, Type expectedType, Func`3 nestedObjectDeserializer, Object& value)
```

I tried with both v3.6.5 and v4.0.0-beta0012 to no avail.
It looks like these properties do not exist in code even though they are documented.

**Comment by ruhullahshah** [comment]
Even though the `assembly-versioning-format` and `assembly-file-versioning-format` are available in the [documentation](http://gitversion.readthedocs.io/en/latest/configuration/), they might not be available in the versions you are using. You can try to use the scheme variant of these variables for now or clone and make a local build.

It works for me. I have a custom build 4.0.0-beta.12+1291.Branch.master.Sha.bc74ac2247d8a58053a33316334ada6b771a8455

**Comment by jbaehr** [comment]
@timoch please try again with the latest version. The feature was merged *after* 4.0.0-beta0013 and, AFAIK *not* back-ported to the v3 line.
